File: SquaresofaSortedArray.java
import java.util.Arrays;

class Solution {
    public int[] sortedSquares(int[] nums) {
    	int size=nums.length;
        int temp[]=new int[size];
        for(int i=0; i<size; i++) {
        	temp[i]=nums[i]*nums[i];

        	
        }
        Arrays.sort(temp);
        return temp;
    }
}
public class SquaresOfASortedArray {
	public static void main(String[] args) {
		Solution soln=new Solution();
		int nums[] = {-4,-1,0,3,10};
		System.out.println(soln.sortedSquares(nums));
	}
}

File: SquaresofaSortedArray.py
class Solution(object):
    def sortedSquares(self, nums):
        """
        :type nums: List[int]
        :rtype: List[int]
        """
        self.nums=nums
        self.arr=[]
        for number in self.nums:
            self.arr.append(number**2)
        self.arr.sort()
        return self.arr
if __name__=="__main__":
    soln=Solution()
    nums = [-4, -1, 0, 3, 10]
    print(soln.sortedSquares(nums))
